app/serializers/project_serializer.rb

```diff
@@ -32,6 +32,7 @@ def self.page(params = {}, scope = nil, context = {})
       elsif params["state"] == "live"
         params["live"] = true
         params.delete("state")
+        scope = Project.where(state: nil)
```

Comment:
the scope is passed into the serializer by the controller and will propogate down, the method signature you've got there is to match the override to inject your behaviour and then call super.